.github/workflows/JirabotMerge.yml

```diff
@@ -221,9 +229,14 @@ jobs:
 
                 result = 'Jirabot Action Result:\n'
 
-                fixVersions = generateFixVersionList(jira, branch_name)
+                fixVersions = generateFixVersionList(jira, project_name, branch_name)
                 result += resolveIssue(jira, issue, fixVersions)
```

Comment:
resolveIssue seems to call for 4 input params, how does this call work?

Reply:
It doesn't I messed that up when merging, fixed now